client/src/commands/toggleFabricRuntimeDevMode.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+import * as vscode from 'vscode';
+import { FabricRuntimeManager } from '../fabric/FabricRuntimeManager';
+import { CommandsUtil } from './commandsUtil';
+import { RuntimeTreeItem } from '../explorer/model/RuntimeTreeItem';
+import { VSCodeOutputAdapter } from '../logging/VSCodeOutputAdapter';
+import { FabricRuntime } from '../fabric/FabricRuntime';
+
+export async function toggleFabricRuntimeDevMode(runtimeTreeItem?: RuntimeTreeItem): Promise<void> {
+    const runtimeManager: FabricRuntimeManager = FabricRuntimeManager.instance();
+    let runtimeName: string;
+    if (!runtimeTreeItem) {
+        runtimeName = await CommandsUtil.showRuntimeQuickPickBox('Enter a name for the runtime');
```

Comment:
surely they are choosing a runtime not giving it a name

Reply:
I've changed the prompt along with all the other instances that you failed to spot in previous reviews 😉